cdhweb/blog/models.py

```diff
@@ -204,6 +204,15 @@ def get_url_parts(self, *args, **kwargs):
             })
             return site_id, root_url, page_path
 
+    def get_sitemap_urls(self, request):
+        """Override sitemap listings to add priority for featured posts."""
+        # output is a list of dict; there should only ever be one element. see:
+        # https://docs.wagtail.io/en/stable/reference/contrib/sitemaps.html#urls
+        urls = super().get_sitemap_urls(request=request)
```

Comment:
seems confusing that this returns a list but there's only ever 1 element, but I see that this is working

Reply:
agree...I guess the use case is a single page that needs multiple sitemap entries, but having trouble imagining why you'd want that.

Reply:
I can — if you generate different formats of the same page/view (similar to text/html/pdf for startwords)